### ISSUE TITLE
Would you add option for cf trace when cf push

### ISSUE THREAD

**Issue by marcus-sds** [patrickcrocker/cf-cli-resource#15]
Would you add option for cf trace when cf push?

CF_TRACE=true

**Comment by patrickcrocker** [comment]
Implemented in 2.4.0-rc.2

Please test and let me know if this works for you!  If so, I'll cut a new release to `latest`.

**Comment by patrickcrocker** [comment]
Did you have an opportunity to test this yet?

**Comment by patrickcrocker** [comment]
all you have to do is specify the `2.4.0-rc.2` version in your pipeline:
```
- name: cf-cli-resource
  type: docker-image
  source:
    repository: pivotalpa/cf-cli-resource
    tag: 2.4.0-rc.2
```

**Comment by patrickcrocker** [comment]
Implemented in v2.4.0. Check the release notes!
